relational_operators/HashJoinOperator.cpp

```diff
@@ -65,21 +65,11 @@ DEFINE_bool(vector_based_joined_tuple_collector, false,
             "If true, use simple vector-based joined tuple collector in "
             "hash join, with a final sort pass to group joined tuple pairs "
             "by inner block. If false, use unordered_map based collector that "
-            "keeps joined pairs grouped by inner block as they are found "
-            "(this latter option has exhibited performance/scaling problems, "
-            "particularly in NUMA contexts).");
+            "keeps joined pairs grouped by inner block as they are found.");
```

Comment:
I don't mind removing it. If I recall correctly, an older profiling done by @pateljm revealed that map collector outperformed the vector collector pretty much always.